File: Warmup1_JAVA/makes10.java
/*
PROMPT
Given 2 ints, a and b, return true if one if them is 10 or
if their sum is 10.
*/

public boolean makes10(int a, int b) {
  return ((a == 10) || (b == 10) || (a + b == 10));
}

/*
This return statement is basically a concentrated version of
three if statements.
It can be expanded into...
---
if (a == 10) {
  return true;
}
else if (b == 10) {
  return true;
}
else if (a + b == 10) {
  return true;
}
return false;
*/

File: Warmup1_PYTHON/makes10.py
'''
PROMPT
Given 2 ints, a and b, return True if one if
them is 10 or if their sum is 10.
'''

def makes10(a, b):
  return (a == 10 or b == 10 or a + b == 10)

'''
This return statement is basically a concentrated version of
three if statements.
It can be expanded into...
---
if (a == 10):
  return True
elif (b == 10):
  return True
elif (a + b == 10):
  return True
return False
'''
